yt_dlp/postprocessor/common.py

```diff
@@ -191,6 +191,16 @@ def report_progress(self, s):
             progress_template.get('postprocess-title') or 'yt-dlp %(progress._default_template)s',
             progress_dict))
 
+        percent = s.get('_percent')
+        if s['status'] not in ('downloading', 'error', 'finished') or percent is None:
+            return
+        # Emit ConEmu progress codes: https://conemu.github.io/en/AnsiEscapeCodes.html#ConEmu_specific_OSC
+        if s['status'] == 'finished':
+            self._downloader._send_console_code('\033]9;4;3;0\007')
+            return
+        state = 1 if s['status'] == 'downloading' else 2
+        self._downloader._send_console_code(f'\033]9;4;{state};{int(percent)}\007')
```

Comment:
I don't like that this code is duplicated. Wouldn't it be better to pass `percent` and `state` to `to_console_title` instead?

Reply:
On the other hand, I dislike adding more functionality to the `to_console_title` function... I do see the point though, as this is specifically tied to `--console-title`. If you think its better covered in there we can do that instead.

Reply:
I'd prefer that. Or atleast a new function. Having the logic in two places seems like a bad idea.

Reply:
If `started` differs, should I just watch for both and pass in the info dict? Or pass the `state` directly as a parameter?

Reply:
`state` makes more sense than `status` imo. Or perhaps `is_error=s['status'] == 'error'`

Reply:
yeah, we should probably apply `3` for postprocessors that do not supply progress?

Reply:
True. Then `state` makes most sense

Reply:
Does something like this work for you?

Reply:
I don't like that `_ProgressState` knows about `_percent`/`status`. But good enough